errcheck/creator/creator.go

```diff
@@ -52,7 +70,35 @@ func Errcheck() checker.Creator {
 						}
 						return okgo.NewIssueFromLine(line, wd)
 					},
-				)), nil
+				),
+			)
+			return &cleanupChecker{c, cleanup}, nil
 		},
 	)
 }
+
+type cleanupChecker struct {
```

Comment:
This is a bit weird, but I didn't see a way in the existing API to run cleanup actions after a check runs.